lib/es6/runtimePaths.js

```diff
@@ -79,7 +79,7 @@ let runtimePaths = {
                 name: "node.lib"
             }],
             tarPath: "node" + "-v" + targetOptions.runtimeVersion + ".tar.gz",
-            headerOnly: false
+            headerOnly: semver.gte(targetOptions.runtimeVersion, '>=4.0.0-alpha')
```

Comment:
That should be semver.satfisfies, because you've included the gte operator in the semver string.

Reply:
Switched to not include the `>=` operator in the semver string